packages/react-composites/tests/app/call/BaseApp.tsx

```diff
@@ -38,9 +38,26 @@ export function BaseApp(props: { queryArgs: QueryArgs; callAdapter?: CallAdapter
     ? () => alert('you are using a unsupported browser')
     : undefined;
 
+  const onPermissionsTroubleshootingClick = queryArgs.useCallReadiness
```

Comment:
Let's rename this `queryArg` to be more specific (perhaps `usePermissionTroubleshootingActions`) to make the connetion more obvious, and rename `useTroubleShootingActions` to `useEnvironmentInfoTroubleshootingOptions`.

"Call Readiness" is not a very meaningful term in itself and the current mapping of these arguments is confusing.